src/plugins/stake-plugins/thorchainSavers/tcSaversPlugin.tsx

```diff
@@ -1220,6 +1221,11 @@ const claimRequest = async (opts: EdgeGuiPluginOptions, request: ChangeQuoteRequ
       utxoSourceAddress,
       forceChangeAddress
     },
+    metadata: {
+      name: 'TCY Claiming',
+      category: 'Income:Claim',
```

Comment:
income should be lower case. This helps it pattern match to translate it.

Reply:
Ok, it's not lowercased anywhere else but I'll change it